service/src/main/java/com/epam/rft/atsy/service/domain/states/AbstractStateDTO.java

```diff
@@ -1,6 +1,7 @@
 package com.epam.rft.atsy.service.domain.states;
```

Comment:
I think this DTO should follow the rename of the State -> StateHistory, so it should be AbstractStateHistoryDTO

With the rename the oddity of having a StateDTO in an AbstractStateDTO would vanish.